Lib/test/test_smtplib.py

```diff
@@ -1393,6 +1393,11 @@ class SMTPUTF8SimTests(unittest.TestCase):
 
     maxDiff = None
 
+    def _is_local(self, date: str) -> bool:
```

Comment:
Open to suggestions on this, in particular I wonder if there's any existing utilities to do something like this ? I think asserting a date is in local format is important for the test case I'm writing to comply with `RFC 5322 section 3.3 4th Paragraph` . refer to https://github.com/python/cpython/pull/136850#discussion_r2217767174

Reply:
yes that's a good alternative, although I would need to also have some cleanup logic similar to that test which is quite a bit of code, it feels to me a little overkill for the purpose in this PR.